### PR TITLE
#203/refactor cleanup

### PR DESCRIPTION
**Pull request by n0macx**
Cleaned and refactored routes, index and few more files.

### CODE REVIEW

File: src/PagesBody/ProfilePage/ProfilePage.js [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
-        text="Twoje Dane"
-        onClick={() => navigate('/userData')}
-        ternary
-      />
-      <MainButton
-        text="Nadchodzące Konkursy"
-        onClick={() => navigate('/upcoming')}
-        secondary
-      />
+      <Link to="/user-dogs">
```
Comment (vieraboschkova) [inline src/PagesBody/ProfilePage/ProfilePage.js:12]: This is a case of button inside an anchor tag, and its semantically incorrect an unnecessary. Perhaps creating a styled link that looks like a button would be an option?
Also @pierwszazlewej consult with the UI/UX designer if the picture of the dog should not appear in another place, it doesn't look doog below the app logo. (probably out of scope of this PR)

**Review by vieraboschkova — CHANGES_REQUESTED** [review]
anchor tag shouldn't contain the button inside, perhaps we should think of a different way to solve it. 
I see kebab-case have been chosen for urls :) Good choice!

File: src/PagesBody/ProfilePage/ProfilePage.js (hunk 1 above)
Comment (n0macx) [reply src/PagesBody/ProfilePage/ProfilePage.js]: Solved

Comment (n0macx) [reply src/PagesBody/ProfilePage/ProfilePage.js]: https://www.youtube.com/watch?v=nf0oXY4nDxE
Link looks like a button

**Review by MonikaKrella — APPROVED** [review]
What a impostor button!

**Review by Smyku6 — APPROVED** [review]
Very nice cleanup Tom!
image

**Pull request merged**